### PR TITLE
Add KSPM recipe for running on ECK

### PR DESCRIPTION
**Pull request by uri-weisman**
Add an example for running a KSPM integration with a standalone agent on ECK.

### CODE REVIEW

**Comment by uri-weisman** [comment]
@thbkrkr - can someone from your team please review my PR?

File: config/recipes/elastic-agent/kspm_integration.yaml [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+            data_stream:
+              dataset: cloud_security_posture.findings
+              type: logs
+            config:
+              v1:
+                type: kspm
+                deployment: self_managed
+                benchmark: cis_k8s
+              processors:
+                - add_cluster_id: null
```
Comment (thbkrkr) [inline config/recipes/elastic-agent/kspm_integration.yaml:550]: Do you have any idea how we will be able to keep this configuration up to date?

**Comment by uri-weisman** [comment]
> I deployed the manifest, connected to `/app/security/cloud_security_posture/dashboard` and it doesn't seem configured?


thank you for the review! have you installed the KSPM integration?

**Comment by thbkrkr** [comment]
> thank you for the review! have you installed the KSPM integration?

Sorry, no. This probably makes sense to you but not to me. Don't take it the wrong way but it's not written anywhere, how should I do this?

People interested in this type of recipe are often people who don't know much about the thing accomplished by the recipe and therefore need to be guided step by step. You can put me in this group. I had to google KSPM to know what it is before reviewing your PR.

We try for all recipes to be self-configured. And if there are prerequisites, we document them (example: [recipes/istio-gateway/README.asciidoc#prerequisites](https://github.com/elastic/cloud-on-k8s/blob/main/config/recipes/istio-gateway/README.asciidoc#prerequisites)). Ideally, it's a few commands to copy/paste in a terminal.

One additional question:  isn't Fleet Server the way to go today to deploy the Agent rather than using standalone?

> We recommend using [Fleet-managed Elastic Agents](https://www.elastic.co/guide/en/fleet/current/install-fleet-managed-elastic-agent.html), when possible, because it makes the management and upgrade of your agents considerably easier.
source: https://www.elastic.co/guide/en/fleet/current/install-standalone-elastic-agent.html

**Comment by thbkrkr** [comment]
ping @uri-weisman

**Pull request closed without merging**